apiserver/src/main/java/org/dependencytrack/util/PageUtil.java

```diff
@@ -0,0 +1,69 @@
+package org.dependencytrack.util;
```

Comment:
Let's move this to:

```suggestion
package org.dependencytrack.persistence.pagination;
```

So we have the pagination stuff in one place.